src/pages/Sponsor.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+  <main class="sponsorship">
+    <h1>寄付者一覧</h1>
+    <p>
+      Twin:teに寄付をしていただき、許可いただいた方を掲載いたします。（敬称略）
+    </p>
+    <p>
+      なお、収支報告書を<a
+        href="https://drive.google.com/drive/folders/1nHj8w5LELC5ZTFnWvgF7HXPriMRovFSp?usp=sharing"
+        style="color: #627fb4"
+        >こちら</a
+      >にて公開しております。
+    </p>
+
+    <div class="sponsor-wrap"></div>
+    <hr />
+
+    <transition name="fade">
+      <p class="amount" v-if="amount != null">
+        総寄付額 <strong>{{ amount }}</strong> 円
+      </p>
+    </transition>
+    <transition name="fade">
+      <p class="count" v-if="count != null">
+        計 <strong>{{ count }}</strong> 人の方に寄付いただきました。
+      </p>
+    </transition>
+
+    <transition name="fade">
+      <div class="users" v-if="count != null && users != null">
+        <a
+          v-for="user in users"
+          :href="user.link"
+          :class="[user.link != null ? 'user-link' : '']"
+          :key="user.displayName"
+          target="_blank"
+        >
+          {{ user.displayName }}
+          <i class="material-icons linkicon">link</i>
+        </a>
+      </div>
+    </transition>
+    <div class="loading" v-if="count == null || users == null">Loading</div>
+    <h2 v-if="count != null && users != null">寄付はこちらから</h2>
+    <a
+      href="https://sponsorship.twinte.net"
+      v-if="count != null && users != null"
+    >
+      <img class="sponsorpage-btn" src="../images/sponsor-ogp.png" />
+    </a>
+  </main>
+</template>
+
+<script lang="ts">
+import { defineComponent, ref } from "vue";
+
+export default defineComponent({
+  name: "Sponsor",
+  setup: () => {
+    const count = ref<number>();
+    const users = ref<any[]>();
+    const amount = ref<number>();
+    fetch("https://api.twinte.net/v3/donation/aggregate/totalAmount")
+      .then((res) => res.json())
+      .then((data) => {
+        amount.value = data.total;
+      });
+    fetch("https://api.twinte.net/v3/donation/aggregate/users")
+      .then((res) => res.json())
+      .then((data) => {
+        users.value = data;
+        count.value = data.length;
+      });
```

Comment:
method not implementsって出るけどこっちで
```suggestion
    const baseUrl = "https://app.twinte.net/api/v3";
    fetch(`${baseUrl}/donation/aggregate/totalAmount`)
      .then((res) => res.json())
      .then((data) => {
        amount.value = data.total;
      });
    fetch(`${baseUrl}/donation/aggregate/users`)
      .then((res) => res.json())
      .then((data) => {
        users.value = data;
        count.value = data.length;
      });
```